renku_notebooks/__init__.py

```diff
@@ -38,16 +36,16 @@ def __init__(self, app):
         self.app = app
 
     def __call__(self, environ, start_response):
-        script_name = environ.get('HTTP_X_SCRIPT_NAME', '')
+        script_name = environ.get("HTTP_X_SCRIPT_NAME", "")
```

Comment:
I'd prefer sticking to single quotes as they have been used in the project so far (also `renku-python` and `renku-gateway` use single quotes).

Reply:
`black` doesn't care what you think! :)

Reply:
here is a nice discussion on the topic: https://github.com/python/black/issues/118

Reply:
I'm not going to be religious about quotes. After spending way too much time configuring `yapf` options, not having a single option to worry about and getting decent-looking code out of it is a win-win for me.

Reply:
Ok, I wasn't aware that `black` was such an opinionated project and I'm a bit astonished that it allows you to configure the line-length but not if you prefer single or double quotes... But fine for me.

Reply:
it allows you to ignore its preference for double-quotes if you wish... but then you end up with inconsistent quotes.